pedrosk-CL.md

```diff
@@ -0,0 +1 @@
+#places to eat
```

Comment:
There was one. on the corner of 176 and OAK
you can go downtown and I think it is called AMORE! (or something like that) next to the tracks